jupyterhub_traefik_proxy/install.py

```diff
@@ -2,26 +2,59 @@
 import hashlib
 import os
 import platform
+import re
 import sys
 import tarfile
 import textwrap
 import warnings
 import zipfile
 from urllib.request import urlretrieve
 
+from packaging.version import parse as parse_version
+
+# BEGIN CHECKSUMS
```

Comment:
Courious about the choice of keeping this in `install.py` and not into their own.

It makes me a little nervous knowing the `update_hashes.py` script modifies this entire file and that an accidental delition of  the `# BEGIN CHECKSUMS` comments will break things.

Reply:
I thought about that! We could definitely put it in its own file, that would make some things easier.